### DISCUSSION TITLE
The mount disk always change name after reboot!

### DISCUSSION THREAD

**Discussion by sinoserendipity** [IceWhaleTech/CasaOS#745] [General]
After I mount a new disk and the disk mount name is Storage1, after reboot, the name change to Storage1-1, after reboot it change to Storage1-1-1, it changes all the time, some application can't start after the mount storage name changed!

**Comment by CZHeron** [comment]
It seems to be fixed in the v0.4.0-alpha1.

**Comment by mr-f0xx** [comment]
I have the same issue smh

**Comment by jfkerekes** [comment]
I am also having this issue. After a restart certain docker apps that rely on a connection to the data drive loose their connection due to the duplication of the drive

**Comment by stevedee78** [comment]
I have the same problem with the last Release.
